lib/taxonomy/publishing_api_root_taxon_parser.rb

```diff
@@ -0,0 +1,27 @@
+module Taxonomy
+  module PublishingApiRootTaxonParser
```

Comment:
Same. The reason we didn't make it a class is to avoid it being instantiated. For now. With more refactoring on the way, I'd be surprised if this sticks around in it's current form for very long.